src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
+Marketplace;marketplace/account;Azure Marketplace;Other;Microsoft;Cloud Provider;Cloud;SaaS
+MFA;mfa/perauthentication;Microsoft Entra ID;Identity;Microsoft;Cloud Provider;Cloud;PaaS
+MFA;mfa/peruser;Microsoft Entra ID;Identity;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AAD;microsoft.aad/domainservices;Azure Active Directory Domain Services;Identity;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AgFoodPlatform;microsoft.agfoodplatform/farmbeats;Azure Data Manager for Agriculture;Integration;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AnalysisServices;microsoft.analysisservices/servers;Azure Analysis Services;Analytics;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.ApiManagement;microsoft.apimanagement/service;API Management;Web;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.apimanagement/service;API Management;Web;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.App;microsoft.app/containerapps;Azure Container Apps;Compute;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.App;microsoft.app/managedenvironments;Azure Container Apps;Compute;Microsoft;Cloud Provider;Cloud;PaaS
+microsoft.appcenter;microsoft.appcenter/accounts;App Center;Mobile;Microsoft;Cloud Provider;Cloud;SaaS
+Microsoft.AppConfiguration;microsoft.appconfiguration/configurationstores;App Configuration;Developer Tools;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.appconfiguration/configurationstores;App Configuration;Developer Tools;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AppPlatform;microsoft.appplatform/spring;Azure Spring Apps;Compute;Microsoft;Cloud Provider;Cloud;PaaS
+microsoft.automation;microsoft.automation/automationaccounts;Azure Automation;Management and Governance;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AutonomousDevelopmentPlatform;microsoft.autonomousdevelopmentplatform/workspaces;Microsoft Autonomous Systems;AI and Machine Learning;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.EventGrid;microsoft.autonomousdevelopmentplatform/workspaces;Microsoft Autonomous Systems;AI and Machine Learning;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AVS;microsoft.avs/privateclouds;Azure VMware Solution;Compute;Microsoft;Cloud Provider;Cloud;IaaS
+Microsoft.AzureActiveDirectory;microsoft.azureactivedirectory/b2cdirectories;Azure Active Directory External Identities;Identity;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AzureActiveDirectory;microsoft.azureactivedirectory/guestusages;Azure Active Directory External Identities;Identity;Microsoft;Cloud Provider;Cloud;PaaS
+Microsoft.AzureArcData;microsoft.azurearcdata/postgresinstances;Azure Arc;Multicloud;Microsoft;Cloud Provider;Both;SaaS
+Microsoft.AzureArcData;microsoft.azurearcdata/sqlmanagedinstances;Azure Arc;Multicloud;Microsoft;Cloud Provider;Both;SaaS
+Microsoft.AzureArcData;microsoft.azurearcdata/sqlserverinstances;Azure Arc;Multicloud;Microsoft;Cloud Provider;On-premises;SaaS

```

Comment:
Instead of "Both", should we use "Hybrid"?
Is it intentional that SQL Server is different than managed instances?

Reply:
corrected